docs/site/Controller-generator.md

```diff
@@ -86,13 +86,12 @@ import {
   del,
   requestBody
 } from '@loopback/rest';
-import {inject} from '@loopback/context';
 import {Todo} from '../models';
 import {TodoRepository} from '../repositories';
 
 export class TodoController {
   constructor(
-    @inject('repositories.TodoRepository')
+    @repository(TodoRepository)
     public todoRepository: TodoRepository,
```

Comment:
Is the type not inferred to be `TodoRepository`?? -- Is there a way for it to be inferred automatically now?

Reply:
That is a very useful feature that I've not thought about. I think this task has been scope creeped enough that I'll create a new issue to implement this feature.